coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/auth/perm/ProposalDevelopmentPermissionsService.java

```diff
@@ -44,4 +44,6 @@
     public void processDeletePermission(ProposalDevelopmentDocument document, ProposalUserRoles proposalUser);
 
     public void processUpdatePermission(ProposalDevelopmentDocument document, ProposalUserRoles proposalUser);
+
+    public String getFullName(String first, String middle, String last);
```

Comment:
this doesn't seem like the best place for this as it doesn't have much to do with permission other than it is used on the permissions page